modules/inflowwind/src/Lidar.f90

```diff
@@ -307,11 +346,25 @@ SUBROUTINE Lidar_CalcOutput( t, u, p, x, xd, z, OtherState, y, m, ErrStat, ErrMs
 
    ErrStat = ErrID_None
    ErrMsg  = ""
+   
+       MeasurementCurrentStep =  INT(t / p%lidar%MeasurementInterval)
+       
+  IF ( (p%lidar%MeasurementInterval * MeasurementCurrentStep) /= t ) THEN
+       Output%VelocityUVW(:,1) = 0 
+   
+  ELSE 
```

Comment:
How about putting a `RETURN` on line 354 so the rest of the function isn't wrapped in the `ELSE` statement. Doing so would also get rid of the mislabeled `ENDIF` on line 644.

Reply:
I've updated the file to include this